docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -3,126 +3,97 @@ title: Connecting to the Internet via USB
 subject: Networking
 ---
 
-
-__Note:__ If you do not need to access the Internet from the EV3 brick, consider
+<span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+If you do not need to access the Internet from the EV3 brick, consider
 using [tethering] instead.
+{: class="alert alert-info"}
 
 [tethering]: /docs/tutorials/using-usb-tethering
 
----
-
-These instructions are for [brickman v0.5.0](http://www.ev3dev.org/news/2015/02/24/Package-Release/).
+<span class="glyphicon glyphicon-warning-sign" aria-hidden="true"></span>
+These instructions are for [brickman v0.7.1](http://www.ev3dev.org/news/2015/02/24/Package-Release/).
```

Comment:
The version number changed but the link to the release didn't? Seems suspicious...